internal/build/env.go

```diff
@@ -87,6 +88,10 @@ func LocalEnv() Environment {
 	if splits := strings.Split(head, " "); len(splits) == 2 {
 		head = splits[1]
 	} else {
+		commitRe, _ := regexp.Compile("^([0-9a-f]{40})$")
```

Comment:
Please add a comment to explain the HEAD file format and why this case is needed. The code is pretty subtle already.

Reply:
Added